<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Format</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>239</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_byteOrder">
      <item>
       <property name="text">
        <string>LittleEndian</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BigEndian</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_channelCount">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_rate">
      <item>
       <property name="text">
        <string>8000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48000</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_sampleSize">
      <property name="text">
       <string>Sample Size (Bit):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_codec">
      <property name="text">
       <string>Codec:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_rate">
      <property name="text">
       <string>Sample Rate (Hz):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_codec">
      <item>
       <property name="text">
        <string>audio/pcm</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_sampleSize">
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_sampleType">
      <property name="text">
       <string>Sample Type:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_byteOrder">
      <property name="text">
       <string>Byte Order:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_channelCount">
      <property name="text">
       <string>Channel Count:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_sampleType">
      <item>
       <property name="text">
        <string>SignedInt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UnSignedInt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Float</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_textMisc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Format settings must match on the sender and receiver side to get
a smooth audio stream.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
